ChatQnA/kubernetes/helm/cpu-values.yaml

```diff
@@ -2,9 +2,22 @@
 # SPDX-License-Identifier: Apache-2.0
 
 vllm:
+  image:
```

Comment:
This value is align with https://github.com/opea-project/GenAIExamples/blob/7072d2cb87a5481e7533c8cb1f9fb63394098446/ChatQnA/docker_compose/intel/cpu/xeon/compose.yaml. You may create another value named, `cpu-perf-values.yaml`.